catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -68,29 +72,50 @@ const SearchForm = ({
     }
   }, [query]);
 
-  function updateUrl(unfilteredSearchResults) {
+  function updateUrl(unfilteredSearchResults, form) {
     const workType = searchParams.workType || [];
     const link = unfilteredSearchResults
       ? worksUrl({
           ...searchParams,
           // Override the defaultWorkType with [] if we're toggled to do so
+          // null => default filters
+          // [] => no filter
+          // [anything] => filter
           workType: workType.length === defaultWorkTypes.length ? [] : workType,
           query: inputQuery,
           page: 1,
+          // $FlowFixMe
+          productionDatesFrom: form.current.productionDatesFrom
+            ? form.current.productionDatesFrom.value
+            : searchParams.productionDatesFrom,
+          // $FlowFixMe
+          productionDatesTo: form.current.productionDatesTo
```

Comment:
I'm a little lost as to when these would be different? Isn't the value always the form value?

Reply:
the filter fields aren't present on work pages or until there is a query